lib/config/fileconf_test.go

```diff
@@ -534,70 +528,96 @@ func TestX11Config(t *testing.T) {
 					"display_offset": math.MaxUint32,
 				}
 			},
-			expectError: require.Error,
 			expectX11Config: &x11.ServerConfig{
 				Enabled:       true,
-				DisplayOffset: x11.DefaultDisplayOffset,
+				DisplayOffset: x11.MaxDisplayNumber,
+				MaxDisplay:    x11.MaxDisplayNumber,
 			},
-		}, {
-			desc: "max displays set",
+		},
+		// Test max display
+		{
+			desc: "max display set",
 			mutate: func(cfg cfgMap) {
 				cfg["ssh_service"].(cfgMap)["x11"] = cfgMap{
-					"enabled":      "yes",
-					"max_displays": "100",
+					"enabled":     "yes",
+					"max_display": "100",
 				}
 			},
-			expectError: require.NoError,
 			expectX11Config: &x11.ServerConfig{
 				Enabled:       true,
 				DisplayOffset: x11.DefaultDisplayOffset,
-				MaxDisplay:    100,
+				MaxDisplay:    100 + x11.DefaultDisplayOffset,
 			},
 		}, {
-			desc: "not enabled, max displays set",
+			// DELETE IN 10.0.0 (Joerger): yaml typo
```

Comment:
Maybe drop the deleter marker? It should cascade from the field deletion.

Reply:
It will only cascade in that this test will start failing since "max_displays" won't resolve to a field. I think leaving this marker will make it more obvious why the test is failing when we go to delete it.